Tests/ttLib/ttVisitor_test.py

```diff
@@ -0,0 +1,18 @@
+from fontTools.ttLib import TTFont
+from fontTools.ttLib.ttVisitor import TTVisitor
+import os
+import pytest
+
+
+class TTVisitorTest(object):
+
+    @staticmethod
+    def getpath(testfile):
+        path = os.path.dirname(__file__)
+        return os.path.join(path, "data", testfile)
+
+    def test_ttvisitor(self):
+
+        font = TTFont(self.getpath("TestVGID-Regular.otf"))
+        visitor = TTVisitor()
+        visitor.visit(font, 1, 2, arg=3)
```

Comment:
it's not immediately clear what this test is actually testing (what are those numbers), especially as I don't see any assertions.

Reply:
It just runs through it, that it doesn't crash. :))

Reply:
The visitor.py is fully tested in the other module. The TTVisitor is just very short module. I'll add more test to it.

Those numbers just test that we can pass arbitrary arguments to the TTVisitor and it passes them through.  I'll add more tests here.